[2025-10-21T20:14:26.611+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-21T20:14:26.634+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airbnb_etl_pipeline.transform_airbnb_forsale manual__2025-10-21T19:57:05.716903+00:00 [queued]>
[2025-10-21T20:14:26.640+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airbnb_etl_pipeline.transform_airbnb_forsale manual__2025-10-21T19:57:05.716903+00:00 [queued]>
[2025-10-21T20:14:26.641+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2025-10-21T20:14:26.651+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_airbnb_forsale> on 2025-10-21 19:57:05.716903+00:00
[2025-10-21T20:14:26.659+0000] {standard_task_runner.py:63} INFO - Started process 419 to run task
[2025-10-21T20:14:26.664+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'airbnb_etl_pipeline', 'transform_airbnb_forsale', 'manual__2025-10-21T19:57:05.716903+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/airbnb_etl_dag.py', '--cfg-path', '/tmp/tmpg85eho6n']
[2025-10-21T20:14:26.667+0000] {standard_task_runner.py:91} INFO - Job 33: Subtask transform_airbnb_forsale
[2025-10-21T20:14:26.717+0000] {task_command.py:426} INFO - Running <TaskInstance: airbnb_etl_pipeline.transform_airbnb_forsale manual__2025-10-21T19:57:05.716903+00:00 [running]> on host b58697e26e05
[2025-10-21T20:14:26.782+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='airbnb_etl_pipeline' AIRFLOW_CTX_TASK_ID='transform_airbnb_forsale' AIRFLOW_CTX_EXECUTION_DATE='2025-10-21T19:57:05.716903+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-21T19:57:05.716903+00:00'
[2025-10-21T20:14:26.784+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-21T20:14:26.792+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-21T20:14:26.793+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/scripts/etl_script.py", line 1215, in transform2
    airdna_data_for_sale = pd.json_normalize(all_properties)
                                             ^^^^^^^^^^^^^^
NameError: name 'all_properties' is not defined
[2025-10-21T20:14:26.811+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=airbnb_etl_pipeline, task_id=transform_airbnb_forsale, run_id=manual__2025-10-21T19:57:05.716903+00:00, execution_date=20251021T195705, start_date=20251021T201426, end_date=20251021T201426
[2025-10-21T20:14:26.823+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 33 for task transform_airbnb_forsale (name 'all_properties' is not defined; 419)
[2025-10-21T20:14:26.849+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-10-21T20:14:26.868+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-21T20:14:26.871+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
